spaceship/spec/portal/tvos_profile_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Spaceship::ProvisioningProfile do
+  describe "Development tvOS Profiles" do
+    before do
+      Spaceship.login
+      adp_enterprise_stubbing
+    end
+    let(:client) { Spaceship::ProvisioningProfile.client }
+
+    describe "Create a new Development tvOS Profile" do
+      it "uses the correct type for the create request" do
+        cert = Spaceship::Certificate::Development.all.first
+        result = Spaceship::ProvisioningProfile::Development.create!(name: 'Delete Me', bundle_id: 'net.sunapps.1', certificate: cert, devices: nil, mac: false, sub_platform: "tvos")
```

Comment:
Can we double-check that "tvos" is the correct capitalization for what is to be passed to Apple? I have the feeling it is supposed to be "tvOS"